### ISSUE TITLE
assembler produces instruction of incorrect size

### ISSUE THREAD

**Issue by bchurchill** [StanfordPL/x64asm#193]
This is a problem if you're trying to generate aligned code.

Right now, I get this:

```
andl $0xe0, %r11d # SIZE=4

Assembling...
andl $0xe0, %r11d


41 81 e3 e0 00 00 00 
```

but I should be getting the output of this:

```
andl $0xe0, %r11d #OPC=132

Assembling...
andl $0xe0, %r11d


41 83 e3 e0
```

**Comment by stefanheule** [comment]
Without having tested anything, I believe this has been fixed in #240.  Re-open if that's not the case.

**Comment by stefanheule** [comment]
Actually, just tested this, and it works correctly now.  The first example doesn't parse (because it shouldn't).  You should use `andl $0xffffffe0, %r11d` instead if you want a 8-bit constant.
